src/analytics/selectors.test.ts

```diff
@@ -9,6 +9,25 @@ describe('getCurrentUserTraits', () => {
       account: { defaultCountryCode: '+33', pincodeType: PincodeType.CustomPin },
       goldToken: { balance: '1.01' },
       stableToken: { balances: { [Currency.Dollar]: '2.02', [Currency.Euro]: '3.03' } },
+      app: {
+        superchargeTokens: [
```

Comment:
We are deprecating this value, we should use `app.superchargeTokenConfigByToken` (see the changes at https://github.com/valora-inc/wallet/pull/2689)